skbio/stats/distance/_base.py

```diff
@@ -508,6 +508,12 @@ def filter(
         MissingIDError
             If an ID in ``ids`` is not in the object's list of IDs.
 
+        Notes
+        -----
+        This function uses parallel computation for improved performance.
```

Comment:
This note looks good. In the future, there might be two improvements: 1) Store this entire sentence somewhere as a constant, which can be re-used in individual documentation pages. This will avoid copying & pasting the same sentence over and over. 2) Some styled text can make it more standing out to the user. For example SciPy [has a table](https://docs.scipy.org/doc/scipy/reference/generated/scipy.cluster.hierarchy.linkage.html) to indicate Python array API support. In our case, the easiest approach is to use the "..note::" box in Sphinx.